src/libos/src/entry.rs

```diff
@@ -87,6 +89,28 @@ pub extern "C" fn occlum_ecall_init(log_level: *const c_char, instance_dir: *con
         unsafe { backtrace::enable_backtrace(&ENCLAVE_PATH, PrintFormat::Short) };
     });
 
+    let resolv_conf_bytes = unsafe {
+        assert!(!resolv_conf_ptr.is_null());
+        CStr::from_ptr(resolv_conf_ptr).to_bytes()
+    };
+    
+    let cfg = match resolv_conf::Config::parse(resolv_conf_bytes) {
```

Comment:
Should copy the string into enclave before using it.

Reply:
Actually, the "CStr::from_ptr" function has copied the pointer and transfers it into slice using slice::from_raw_parts.